cluster-autoscaler/cloudprovider/aws/auto_scaling_groups.go

```diff
@@ -23,10 +23,9 @@ import (
 	"sync"
 	"time"
 
+	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider/aws/aws-sdk-go/aws"
+	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider/aws/aws-sdk-go/service/autoscaling"
```

Comment:
I had to change all the imports to use the vendored SDK, but only changing some would make the tests fail, so I had to change all of them.